src/zosmf/setup.py

```diff
@@ -1,8 +1,21 @@
+import os.path
 import sys
+import uuid
 from setuptools import setup, find_namespace_packages
 sys.path.append("..")
 from _version import __version__
 
+src_dir = os.path.realpath(os.path.dirname(os.path.dirname(__file__)))
+
+def resolve_sdk_dep(sdk_name, version_spec):
+    if os.path.exists(os.path.join(src_dir, sdk_name, "zowe")):
+        # Handle building from a Git checkout
+        # Based on https://github.com/lab-cosmo/equistore/blob/master/python/equistore-torch/setup.py#L212
+        sdk_dir = os.path.realpath(os.path.join(src_dir, sdk_name))
+        return f"zowe.{sdk_name}_for_zowe_sdk@file://{sdk_dir}?{uuid.uuid4()}"
+    else:
+        return f"zowe.{sdk_name}_for_zowe_sdk{version_spec}"
```

Comment:
Can this be separated into a common script/file that we can reference in all the `setup.py`'s?